test/integration_test.py

```diff
@@ -995,49 +994,52 @@ def _file_ext(self, file: JSON) -> str:
         suffixes = PurePath(file['name']).suffixes
         return ''.join(suffixes).lower()
 
-    def _validate_file_content(self, content: ReadableFileObject, file_ext: str):
+    def _validate_file_content(self, content: ReadableFileObject, file: JSON):
+        file_ext = self._file_ext(file)
         if file_ext == '.fastq':
             self._validate_fastq_content(content)
         elif file_ext == '.fastq.gz':
             with gzip.open(content) as buf:
                 self._validate_fastq_content(buf)
-        else:
+        elif file['size'] > 0:
             self.assertEqual(1, len(content.read(1)))
+        else:
+            log.info('File %r is empty', file['uuid'])
```

Comment:
You should assert that this is indeed the case and then we won't need the log statement.